examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3237,6 +3237,195 @@ server cluster ActivatedCarbonFilterMonitoring = 114 {
   command ResetCondition(): DefaultSuccess = 0;
 }
 
+/** This cluster provides an interface to the functionality of Smart Energy Demand Response and Load Control. */
+provisional server cluster DemandResponseLoadControl = 150 {
+  enum CriticalityLevelEnum : enum8 {
+    kUnknown = 0;
```

Comment:
```suggestion
    kInvalid = 0;
```
it is Invalid on the specs not unknow

Reply:
Please see PR description: this is targeting an outstanding spec PR, not the already-merged spec.